lms/static/sass/views/_bookmarks.scss

```diff
@@ -0,0 +1,144 @@
+// Rules for placing bookmarks and search button side by side
+.wrapper-course-modes {
+  border-bottom: 1px solid $gray-l3;
+  display: table;
+  width: 98%;
+  padding: 6px 3px;
+
+  > div {
+    display: table-cell;
+  }
+}
+
+
+// Rules for Bookmarks Button
+.courseware-bookmarks-button {
+  width: 40%;
+  padding: 0 3px;
+  vertical-align: top;
```

Comment:
I was able to remove vertical-align: top without any discernable impact given the above changes. If that's possible, remove this line.